src/server/qgsaccesscontrol.h

```diff
@@ -0,0 +1,54 @@
+/***************************************************************************
+                              qgsaccesscontrol.h
+                              ------------------
+  begin                : 22-05-2015
+  copyright            : (C) 2008 by Stéphane Brunner
+  email                : stephane dot brunner at camptocamp dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSACCESSCONTROL_H
+#define QGSACCESSCONTROL_H
+
+#include "qgsextraexpressionprovider.h"
+#include "qgsaccesscontrolplugin.h"
+
+#include <QMultiMap>
+
+class QgsAccessControlPlugin;
+
+
+/** \ingroup core
```

Comment:
\ingroup server ? This does not exists but it will be eventually created for the server API